broker/src/main/java/org/apache/rocketmq/broker/processor/PopBufferMergeService.java

```diff
@@ -536,6 +537,12 @@ private void putCkToStore(final PopCheckPointWrapper pointWrapper, final boolean
             POP_LOGGER.error("[PopBuffer]put ck to store fail: {}, {}", pointWrapper, putMessageResult);
             return;
         }
+        
+        this.brokerController.getBrokerStatsManager().incTopicPutNums(TopicValidator.RMQ_SYS_SCHEDULE_TOPIC, putMessageResult.getAppendMessageResult().getMsgNum(), 1);
```

Comment:
And it's duplicated codes, it's better to put these codes into a new method in BrokerStatsManager?